### DISCUSSION TITLE
感谢您的作品！希望能出一个漫画缩放比例记忆功能

### DISCUSSION THREAD

**Discussion by OthuMMMM** [tonquer/picacg-qt#293] [General]
感谢您的辛劳付出，给我们呈现了这样一个实用方便又伟大的作品！不过作为习惯使用上下滚动阅读模式的人，每次打开漫画/切换窗口后都需要重新调节缩放比例稍微有点不方便，非常希望您能够出一个记忆漫画缩放比例的功能。如果您有意愿且有空闲的情况下能够实现这个功能就太感谢了。再次感谢您提供的优秀作品，非常感谢！

**Comment by tonquer** [comment]
是重新打开软件要重新调吗，重新换一本漫画，缩放应该是会保存的
